### ISSUE TITLE
iam.createUser fails with TypeError "Cannot read property 'member' of undefined"

### ISSUE THREAD

**Issue by trivikr** [aws/aws-sdk-js-v3#765]
**Describe the bug**
iam.createUser doesn't create user

**SDK version number**
```console
$ yarn list aws-sdk
└─ aws-sdk@2.605.0
$ yarn list @aws-sdk/client-iam
└─ @aws-sdk/client-iam@1.0.0-alpha.2
```

**Is the issue in the browser/Node.js?**
Node.js

**Details of the browser/Node.js version**
v12.14.0

**To Reproduce (observed behavior)**
Reproduced with the following code:
```javascript
const AWS = require("aws-sdk");
const { IAM } = require("@aws-sdk/client-iam");

(async () => {
  let response;
  const region = "us-west-2";
  const UserName =
    "test-user-" +
    Math.random()
      .toString(36)
      .substring(2, 15);

  const v2Client = new AWS.IAM({ region });
  response = await v2Client.createUser({ UserName }).promise();
  console.log("Data returned by v2:");
  console.log(response);
  await v2Client.waitFor("userExists", { UserName }).promise();
  await v2Client.deleteUser({ UserName }).promise();

  const v3Client = new IAM({ region });
  try {
    await v3Client.createUser({ UserName });
  } catch (e) {
    console.log("\nError returned by v3:");
    console.log(e);
  }
})();
```

Output:
```console
Data returned by v2:
{
  ResponseMetadata: { RequestId: 'e0b645a3-6c3f-4208-bc8c-48966258f45f' },
  User: {
    Path: '/',
    UserName: 'test-user-c105b0nx29m',
    UserId: 'AIDA5ROEOH3WFYCUZYQ7N',
    Arn: 'arn:aws:iam::930809265900:user/test-user-c105b0nx29m',
    CreateDate: 2020-01-21T00:59:20.000Z,
    Tags: []
  }
}

Error returned by v3:
TypeError: Cannot read property 'member' of undefined
    at deserializeAws_queryUser (/Users/trivikr/workspace/AwsJavaScriptSdkIntegTests/node_modules/@aws-sdk/client-iam/dist/cjs/protocols/Aws_query.js:10792:23)
    at deserializeAws_queryCreateUserResponse (/Users/trivikr/workspace/AwsJavaScriptSdkIntegTests/node_modules/@aws-sdk/client-iam/dist/cjs/protocols/Aws_query.js:8449:25)
    at Object.deserializeAws_queryCreateUserCommand (/Users/trivikr/workspace/AwsJavaScriptSdkIntegTests/node_modules/@aws-sdk/client-iam/dist/cjs/protocols/Aws_query.js:2036:16)
    at processTicksAndRejections (internal/process/task_queues.js:93:5) {
  '$metadata': { retries: 0, totalRetryDelay: 0 }
}
```

**Expected behavior**
iam.createUser should create user ([docs](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/IAM.html#createUser-property))

**Comment by trivikr** [comment]
cloudformation.describeStacks() also fails with error
```console
Cannot read property 'member' of undefined
```

Tested with the following JavaScript code:
```javascript
const AWS = require("aws-sdk");
const { CloudFormation } = require("@aws-sdk/client-cloudformation");

(async () => {
  let response;
  const region = "us-west-2";

  const v2Client = new AWS.CloudFormation({ region });
  response = await v2Client.describeStacks().promise();
  console.log("Data returned by v2:");
  console.log(response);

  const v3Client = new CloudFormation({ region });
  try {
    await v3Client.describeStacks({});
  } catch (e) {
    console.log("\nError returned by v3:");
    console.log(e);
  }
})();
```

Output:
```console
Data returned by v2:
{
  ResponseMetadata: { RequestId: 'a503de28-9855-4f29-9b18-dbe9971b1adf' },
  Stacks: [ <STACKS> ]
}

Error returned by v3:
TypeError: Cannot read property 'member' of undefined
    at deserializeAws_queryRollbackConfiguration (/Users/trivikr/workspace/AwsJavaScriptSdkIntegTests/node_modules/@aws-sdk/client-cloudformation/dist/cjs/protocols/Aws_query.js:4587:35)
    at deserializeAws_queryStack (/Users/trivikr/workspace/AwsJavaScriptSdkIntegTests/node_modules/@aws-sdk/client-cloudformation/dist/cjs/protocols/Aws_query.js:4688:42)
    at /Users/trivikr/workspace/AwsJavaScriptSdkIntegTests/node_modules/@aws-sdk/client-cloudformation/dist/cjs/protocols/Aws_query.js:5071:42
    at Array.map (<anonymous>)
    at deserializeAws_queryStacks (/Users/trivikr/workspace/AwsJavaScriptSdkIntegTests/node_modules/@aws-sdk/client-cloudformation/dist/cjs/protocols/Aws_query.js:5071:27)
    at deserializeAws_queryDescribeStacksOutput (/Users/trivikr/workspace/AwsJavaScriptSdkIntegTests/node_modules/@aws-sdk/client-cloudformation/dist/cjs/protocols/Aws_query.js:3920:27)
    at Object.deserializeAws_queryDescribeStacksCommand (/Users/trivikr/workspace/AwsJavaScriptSdkIntegTests/node_modules/@aws-sdk/client-cloudformation/dist/cjs/protocols/Aws_query.js:896:16)
    at processTicksAndRejections (internal/process/task_queues.js:93:5) {
  '$metadata': { retries: 0, totalRetryDelay: 0 }
}
```

```console
$ yarn list aws-sdk
└─ aws-sdk@2.605.0
$ yarn list @aws-sdk/client-cloudformation
└─ @aws-sdk/client-cloudformation@1.0.0-alpha.2
```

**Comment by trivikr** [comment]
@kstich is working on fixing this issue

**Comment by trivikr** [comment]
Confirmed that the IAM issue is fixed in `@aws-sdk/client-iam@1.0.0-alpha.3`

Verified using code:
```javascript
const AWS = require("aws-sdk");
const { IAM } = require("@aws-sdk/client-iam");

(async () => {
  let response;
  const region = "us-west-2";
  const UserName =
    "test-user-" +
    Math.random()
      .toString(36)
      .substring(2, 15);

  const v2Client = new AWS.IAM({ region });
  response = await v2Client.createUser({ UserName }).promise();
  console.log("Data returned by v2:");
  console.log(response);
  await v2Client.waitFor("userExists", { UserName }).promise();
  await v2Client.deleteUser({ UserName }).promise();

  const v3Client = new IAM({ region });
  const v3UserName = `${UserName}V3`;
  response = await v3Client.createUser({ UserName: v3UserName });
  console.log("\nData returned by v3:");
  console.log(response);
  await v2Client.waitFor("userExists", { UserName: v3UserName }).promise();
  await v2Client.deleteUser({ UserName: v3UserName }).promise();
})();
```

Example output:
```console
Data returned by v2:
{
  ResponseMetadata: { RequestId: 'fa34baff-1b52-43ff-90a1-c2715d2e6c35' },
  User: {
    Path: '/',
    UserName: 'test-user-w5bi7ra6uan',
    UserId: 'AIDA5ROEOH3WEYO4JJJ7F',
    Arn: 'arn:aws:iam::930809265900:user/test-user-w5bi7ra6uan',
    CreateDate: 2020-01-22T01:51:05.000Z,
    Tags: []
  }
}

Data returned by v3:
{
  '$metadata': {
    httpStatusCode: 200,
    httpHeaders: {
      'x-amzn-requestid': 'a15ec92d-9284-426f-8256-011b7e58dd8f',
      'content-type': 'text/xml',
      'content-length': '507',
      date: 'Wed, 22 Jan 2020 01:51:06 GMT'
    },
    requestId: 'a15ec92d-9284-426f-8256-011b7e58dd8f',
    retries: 0,
    totalRetryDelay: 0
  },
  __type: 'CreateUserResponse',
  User: {
    __type: 'User',
    Arn: 'arn:aws:iam::930809265900:user/test-user-w5bi7ra6uanV3',
    CreateDate: 2020-01-22T01:51:06.000Z,
    PasswordLastUsed: undefined,
    Path: '/',
    PermissionsBoundary: undefined,
    Tags: undefined,
    UserId: 'AIDA5ROEOH3WOSQQX3UF4',
    UserName: 'test-user-w5bi7ra6uanV3'
  }
}
```

**Comment by trivikr** [comment]
Verified that the issue is fixed in `@aws-sdk/client-cloudformation@1.0.0-alpha.3`

Code:
```javascript
const AWS = require("aws-sdk");
const { CloudFormation } = require("@aws-sdk/client-cloudformation");

(async () => {
  let response;
  const region = "us-west-2";

  const v2Client = new AWS.CloudFormation({ region });
  response = await v2Client.describeStacks().promise();
  console.log("Data returned by v2:");
  console.log(response);

  const v3Client = new CloudFormation({ region });
  response = await v3Client.describeStacks({});
  console.log("\nData returned by v3:");
  console.log(response);
})();

```

Output:
```console
Data returned by v2:
{
  ResponseMetadata: { RequestId: 'a503de28-9855-4f29-9b18-dbe9971b1adf' },
  Stacks: [ <STACKS> ]
}

Data returned by v3:
{
  '$metadata': {
    httpStatusCode: 200,
    httpHeaders: {
      'x-amzn-requestid': 'e41a372f-b477-4e5d-b8d0-4dbfa5d21d09',
      'content-type': 'text/xml',
      'content-length': '22075',
      vary: 'accept-encoding',
      date: 'Wed, 22 Jan 2020 01:54:58 GMT'
    },
    requestId: 'e41a372f-b477-4e5d-b8d0-4dbfa5d21d09',
    retries: 0,
    totalRetryDelay: 0
  },
  __type: 'DescribeStacksOutput',
  NextToken: undefined,
  Stacks: [ <STACKS> ]
}
```
